crates/masonry/src/render_root.rs

```diff
@@ -144,6 +162,20 @@ impl RenderRoot {
         self.root_on_text_event(event)
     }
 
+    // TODO: Is this the right place for this?
+    pub fn add_test_font(
```

Comment:
Seems reasonable. RenderRoot is the type ultimately responsible for turning semantic data into bitmaps. You can either pass a test font with a constructor argument, a build method, or a setter. I think a setter is fine.